frontend/app/vite.config.ts

```diff
@@ -84,6 +84,28 @@ export default defineConfig({
   server: {
     open: true,
     port: 3000,
+    host: true,
+    proxy: {
+      // You can find all backend endpoints in
+      // https://github.com/streamlit/streamlit/blob/develop/lib/streamlit/web/server/server.py
```

Comment:
Should we put a comment in `server.py` to add a new endpoint here?

Reply:
Good point 👍 Added comments to `server.py` and `DefaultStreamlitEndpoints.ts` to keep these endpoints in sync.